tests/components/plex/test_config_flow.py

```diff
@@ -502,3 +530,194 @@ async def test_multiple_servers_with_import(hass):
+            result["flow_id"], user_input=MANUAL_SERVER
+        )
+
+    assert result["type"] == "create_entry"
+    assert result["title"] == mock_plex_server.friendlyName
+    assert result["data"][CONF_SERVER] == mock_plex_server.friendlyName
+    assert result["data"][CONF_SERVER_IDENTIFIER] == mock_plex_server.machineIdentifier
+    assert result["data"][PLEX_SERVER_CONFIG][CONF_URL] == mock_plex_server._baseurl
+    assert result["data"][PLEX_SERVER_CONFIG][CONF_TOKEN] == MOCK_TOKEN
+
+
+async def test_manual_config_with_token(hass):
+    """Test creating via manual configuration with only token."""
+
+    result = await hass.config_entries.flow.async_init(
+        config_flow.DOMAIN, context={"source": "user", "show_advanced_options": True}
+    )
+
+    assert result["type"] == "form"
+    assert result["step_id"] == "user_advanced"
+
+    result = await hass.config_entries.flow.async_configure(
+        result["flow_id"], user_input={"setup_method": MANUAL_SETUP_STRING}
+    )
+
+    assert result["type"] == "form"
+    assert result["step_id"] == "manual_setup"
+
+    mock_plex_server = MockPlexServer()
+
+    with patch("plexapi.myplex.MyPlexAccount", return_value=MockPlexAccount()), patch(
+        "plexapi.server.PlexServer", return_value=mock_plex_server
+    ), patch("homeassistant.components.plex.PlexWebsocket.listen"):
+        result = await hass.config_entries.flow.async_configure(
+            result["flow_id"], user_input={CONF_TOKEN: MOCK_TOKEN}
+        )
+
+    assert result["type"] == "create_entry"
+    assert result["title"] == mock_plex_server.friendlyName
+    assert result["data"][CONF_SERVER] == mock_plex_server.friendlyName
+    assert result["data"][CONF_SERVER_IDENTIFIER] == mock_plex_server.machineIdentifier
+    assert result["data"][PLEX_SERVER_CONFIG][CONF_URL] == mock_plex_server._baseurl
+    assert result["data"][PLEX_SERVER_CONFIG][CONF_TOKEN] == MOCK_TOKEN
+
+
+async def test_setup_with_limited_credentials(hass):
+    """Test setup with a user with limited permissions."""
+    mock_plex_server = MockPlexServer()
+
+    entry = MockConfigEntry(
+        domain=DOMAIN,
+        data=DEFAULT_DATA,
+        options=DEFAULT_OPTIONS,
+        unique_id=DEFAULT_DATA["server_id"],
+    )
+
+    with patch(
+        "plexapi.server.PlexServer", return_value=mock_plex_server
+    ), patch.object(
+        mock_plex_server, "systemAccounts", side_effect=plexapi.exceptions.Unauthorized
+    ) as mock_accounts, patch(
+        "homeassistant.components.plex.PlexWebsocket.listen"
+    ) as mock_listen:
+        entry.add_to_hass(hass)
+        assert await hass.config_entries.async_setup(entry.entry_id)
+        await hass.async_block_till_done()
+
+    assert mock_listen.called
+    assert mock_accounts.called
+
+    plex_server = hass.data[DOMAIN][SERVERS][mock_plex_server.machineIdentifier]
+    assert len(plex_server.accounts) == 0
+    assert plex_server.owner is None
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5462445)